chaoslib/litmus/container_kill/containerd_chaos/containerd.j2

```diff
@@ -32,4 +32,4 @@ spec:
           name: cri-socket
         - hostPath:
             path: /etc/crictl.yml 
-          name: cri-config
+          name: cri-config
```

Comment:
Can you please convert `path: /etc/crictl.yml ` to `path: /etc/crictl.yaml ` at all places in thiis file?

Reply:
Done